app/Http/Controllers/AuditLogController.php

```diff
@@ -44,6 +45,9 @@ public function index(Request $request)
         if ($listDetails['date_to']) {
             $query->where('created_at', '<=', $listDetails['date_to']);
         }
+        if ($listDetails['ip']) {
+            $query->where('ip', '=', $listDetails['ip']);
```

Comment:
I think it would prove useful if this was a prefix-based search so, for example, you could search based on the first 3 parts of an IP (`127.0.0.`) in the event that someone was searching for an IP range.

Reply:
IP address is save as string in database. It is good to go searching with a prefix.

If we want to provide searching with IPv4 / IPv4 and CIDR, it is better to save data in integer.